qml/Aqt/Cassowary/qmldir.in

```diff
@@ -1,4 +1,4 @@
 module Aqt.Cassowary
 
 Stay 1.0 Stay.qml
-plugin AqtCassowaryPlugin @PATH_TO_PLUGIN@
+plugin AqtCassowaryPlugin
```

Comment:
@nre-ableton: this file must be renamed to `qmldir` (without the `.in` extension)